docs/faq/faq.md

```diff
@@ -59,6 +59,31 @@ def wait_on_workflow(*args):
     )
     sensor.execute()
 ```
+
+## How do I wait for a specific task in a workflow to finish before kicking off my own workflow's tasks?
+
+```python
+from brickflow.context import ctx
+from brickflow_plugins import WorkflowTaskDependencySensor
+
+wf = Workflow(...)
+
+
+@wf.task
+def wait_on_workflow(*args):
+    api_token_key = ctx.dbutils.secrets.get("scope", "api_token_key")
+    sensor = WorkflowTaskDependencySensor(
+        databricks_host="https://your_workspace_url.cloud.databricks.com",
+        databricks_token=api_token_key,
+        dependency_job_id=job_id,
```

Comment:
There have been some issues with using `job_id` even in WorkflowDependencySensor, as the job_id changes continuously when there are any changes or redeployments happening to the job. It might be best to use job_name and internally fetch the `job_id` using a common function. Let's park it for next issue?

Reply:
Good point, using `job_name` instead, will be much more user friendly. I can incorporate the code to fetch that into this PR as well, won't take a lot of time.
Could be another method of the `WorkflowDependencySensor` class, or if you feel it might be reused somewhere else, it can also go to `brickflow_plugins/databricks/__init__.py`. Let me know.

Reply:
Let's have it in [here](https://github.com/Nike-Inc/brickflow/blob/main/brickflow/engine/utils.py), there are other use cases where we would need to get job id as well. Example - `RunJobTask`

Reply:
Done. Please review, when you have time.